Makefile

```diff
@@ -192,7 +187,7 @@ $(BUILD_DIRECTORY)/electron-$(TARGET_PLATFORM)-$(APPLICATION_VERSION)-$(TARGET_A
 	./scripts/build/electron-create-resources-app.sh -s . -o $@ \
 		-v $(APPLICATION_VERSION) \
 		-f "$(APPLICATION_FILES)"
-	$(foreach prerequisite,$^,$(call execute-command,cp -rf $(prerequisite) $@))
+	$(foreach prerequisite,$^,$(call execute-command,cp -RLrf $(prerequisite) $@))
```

Comment:
Interesting. My BSD implementation of `cp` claims that `-r` is for backwards compatibility purposes and that is similar to `-R`, but not identical. Anyway, let's try just `-R` and see how it goes on the build servers.